zerver/context_processors.py

```diff
@@ -131,6 +131,7 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         'settings_comments_path': settings_comments_path,
         'platform': platform,
         'allow_search_engine_indexing': allow_search_engine_indexing,
+        'max_file_upload_size': settings.MAX_FILE_UPLOAD_SIZE,
```

Comment:
This should go in `zerver/views/home.py`, not here, since it is only needed in the logged-in app.

Reply:
I made the changes to `zerver/views/home.py` in the previous PR that was merged. The conditions I added in the html templates didn't work until I made this change.

Reply:
Add it to the `response = render(request, 'zerver/app/index.html',` line (copying out of `page_params`); indeed it needs to be in the `context` for that page, we just don't need it in the context for every page :).

Reply:
Got it, and made the changes. Thank you. :)